pytorch/ignite
# [#2325] Remove old docs related at https://pytorch.org/ignite/

Fixes #2325

Description: as discussed in #2325 we want to delete all old content that's not hosted on the new domain.

I've deleted all content that was replaced with the new domain and kept only the API reference docs as I couldn't find them in the new domain.

**This is a WIP to use as a discussion forum till we agree on the optimal new design for the old doc page**

To test it
```
1. cd docs
2. make html
3. cd build/
4. python -m http.server 8989
```
This is how it looks like

<img width="1440" alt="Screen Shot 2022-01-18 at 11 59 39 PM" src="https://user-images.githubusercontent.com/2458523/150038219-36f518c8-ccb2-44e1-ba1e-9046a571cb5a.png">



Checked the links and they referencing the updated links instead of the old html.

**I didn't change the main URL (https://pytorch.org/ignite')  to the new URL because it will break the pages not available for the new URL (for instance: examples), will change the base to the new URL once all of the URLs are available on the (pytorch-ignite.ai/) URL**

Check list:

- [x] New tests are added (if a new feature is added)
- [x] New doc strings: description and/or example code are in RST format
- [x] Documentation is updated (if required)
